zookeeper/conf/zookeeper.init.d

```diff
@@ -31,7 +31,7 @@ desc="ZooKeeper daemon"
 
 start() {
   echo -n $"Starting $desc (zookeeper): "
-  daemon --user zookeeper ${SERVER_SCRIPT} start
+  daemon --user {{ zk.user }} ${SERVER_SCRIPT} start
```

Comment:
Seems that `zk` dictionary context will not be passed to this template at all.
I think you should explicitly pass the `user` variable the same way it works for `alt_home`.
Same applies to `zookeeper.init.d.ubuntu`.

Reply:
Hi @vutny,

please see my last commit, this should fix the zk user in init scripts.